README.md

```diff
@@ -8,11 +8,47 @@ simulation input data representing the past nuclear fuel cycle in the united
 states. It is being conducted by undergraduate researchers at the University of
 Illinois at Urbana-Champaign in the Advanced Reactors and Fuel Cycles group.
 This work benefits enormously from previous work by Robert Flanagan in the
-fleetcomp repository.  
+fleetcomp repository.
 
 Not Ready For Reuse
 ====================
 
 During development, this work is under sole development by the authors in
-preparation for publication in conference proceedings or an archival journal. 
+preparation for publication in conference proceedings or an archival journal.
 
+Generating Cycamore Reactor input
+=================================
+
+Cycamore reactor input and its in_commod and out_commod recipies are produced
+from the 'US_Fleet.txt' file and 'vision_recipes.xls' file. This also serves
+as an example of xml inclusion within an xml file using xinclude. Xinclude
+will need to be processed by an xml parser such as the xml_parser.py script
+in the repository.
+
+Usage:
+1. python recipe_generator.py [InputData] [TemplateFile]
+2. ~~python xml_parser.py~~ (This is only for demonstrating xml inclusion)
```

Comment:
I'm still not sure I understand. If it's not useful or shouldn't run then it probably doesn't need to be in the repository.